resources/css/app.css

```
@@ -3,6 +3,7 @@ html {
     --font-family: "Geist";
 
     scroll-behavior: smooth;
+    interpolate-size: allow-keywords;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)